css/properties/break-inside.json

```
@@ -118,7 +118,7 @@
               "opera": "mirror",
               "opera_android": "mirror",
               "safari": {
-                "version_added": "≤13.1"
+                "version_added": "10"
```

According to https://bugs.webkit.org/show_bug.cgi?id=61096#c6, `-webkit-column-break-inside: avoid` was supported:

```suggestion
                "version_added": "3"
```

---

This change causes the consistency linter to fail, and we record `-webkit-column-break-inside` as a separate property in BCD, so let's keep it as "10" in this file!